### PR TITLE
fix simple type error.

### PR DESCRIPTION
**Pull request by soarpenguin**

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -119,13 +119,12 @@ func (a *App) Run(arguments []string) (err error) {
...
 	err = set.Parse(arguments[1:])
 	nerr := normalizeFlags(a.Flags, set)
+	context := NewContext(a, set, nil)
 	if nerr != nil {
 		fmt.Fprintln(a.Writer, nerr)
-		context := NewContext(a, set, nil)
 		ShowAppHelp(context)
 		return nerr
 	}
-	context := NewContext(a, set, nil)
```
Comment (jszwedko) [inline app.go]: I actually just merged #329 which had this same change.

File: command.go [hunk 2]
```diff
@@ -189,7 +189,7 @@ func (c Command) startApp(ctx *Context) error {
 
 	// set the name and usage
 	app.Name = fmt.Sprintf("%s %s", ctx.App.Name, c.Name)
-	if c.HelpName == "" {
+	if c.HelpName != "" {
```
Comment (jszwedko) [inline command.go]: Oops, that is definitely a bug.

Comment (jszwedko) [inline command.go]: Actually, I take that back, that logic appears to be intentional -- was it manifesting in an issue for you?

**Comment by meatballhat** [comment]
@soarpenguin Please yell if you're still having issues related to this 😺

**Pull request closed without merging**